### PR TITLE
Sally

### PR DESCRIPTION
**Pull request by SallyEllisGitHub**
Can you review this

### CODE REVIEW

File: individual files/sally.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a test
```
Comment (cyrilgagnaire) [inline individual files/sally.txt:1]: Hey can you change to a capital T

**Review by cyrilgagnaire — CHANGES_REQUESTED** [review]

File: individual files/sally.txt (hunk 1 above)
Comment (SallyEllisGitHub) [reply individual files/sally.txt]: Done

Comment (cyrilgagnaire) [reply individual files/sally.txt]: Thanks

**Review by cyrilgagnaire — APPROVED** [review]
Good job

**Pull request merged**